SAMLPortal/Client/src/css/site.css

```diff
@@ -0,0 +1,137 @@
+﻿/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+for details on configuring this project to bundle and minify static web assets. */
+
+a.navbar-brand {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Element (a.navbar-brand) is overqualified, just use .navbar-brand without element name.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5434886)